x/evm/keeper/migrate.go

```diff
@@ -49,6 +61,102 @@ func removeExternalTokenBurnerCode(ctx sdk.Context, ck chainKeeper) error {
 	return nil
 }
 
+func migrateCommandBatchSignature(ctx sdk.Context, ck chainKeeper, signer types.Signer, multisig types.MultisigKeeper) error {
+	commandBatchID := ck.getLatestSignedCommandBatchID(ctx)
+	for {
+		commandBatchMetadata := ck.getCommandBatchMetadata(ctx, commandBatchID)
+		if commandBatchMetadata.ID == nil {
+			break
+		}
+		commandBatchID = commandBatchMetadata.PrevBatchedCommandsID
+
+		if commandBatchMetadata.Status != types.BatchSigned {
+			if commandBatchMetadata.Status == types.BatchSigning {
+				setCommandBatchAborted(ctx, ck, commandBatchMetadata)
+			}
+			continue
+		}
+
+		key, ok := multisig.GetKey(ctx, commandBatchMetadata.KeyID)
+		if !ok {
+			break
+		}
+
+		sigID := hex.EncodeToString(commandBatchMetadata.ID)
+		multisigInfo, found := signer.GetMultisigSignInfo(ctx, sigID)
+		signInfo, ok := multisigInfo.(*tsstypes.MultisigInfo)
```

Comment:
Comparing this to the tss migrate at https://github.com/axelarnetwork/axelar-core/blob/9da27e49358263ad76a4bbfd95cf5ce5c4415a9e/x/tss/keeper/migrate.go#L107, are we using the same type `tsstypes.MultisigInfo` to represent both multisig key and signature in tss?!

Reply:
no, there are two very similar sounding names, is this a casting error?

Reply:
yes, from what I recapped in the migration. write in same type, load by different interfaces.